cpp_test/ImageCap.cpp

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+#include <opencv2/opencv.hpp>
+
+using namespace cv;
+using namespace std;
+
+class ImageCap
+{
+    private:
+        Mat frame;
+
+    public:
+        ImageCap(int port)
+        {
+            capImage(port);
+        }
+
+        void capImage(int port)
+        {
+            VideoCapture cap(port);
+
+            bool capSuccess = cap.read(frame);
```

Comment:
Doesn't there need to be some error checking here in the event the VideoCapture fails? It may not actually get a port, and then you'll likely get an exception here.